lib/openactive/dataset_site/settings.rb

```diff
@@ -94,6 +102,20 @@ def booking_service
         )
       end
 
+      def access_service
+        OpenActive::Models::WebAPI.new(
+          name: 'Open Booking API',
+          description: "API that allows for seamless booking experiences to be created for #{data_feed_descriptions.to_sentence} available from #{organisation_name}".downcase,
```

Comment:
```suggestion
          description: "API that allows for seamless booking experiences to be created for #{data_feed_descriptions.to_sentence.downcase} available from #{organisation_name}",
```

Reply:
Would this work instead? It's only the `data_feed_descriptions.to_sentence` that needs to be lower cased...